tests/conftest.py

```diff
@@ -22,13 +23,26 @@ def pytest_addoption(parser):
 
 
 @pytest.fixture(scope="session")
+# instantiates ini file parses object
 def prep_properties():
     config_reader = ConfigParserIni("props.ini")
     return config_reader
 
 
+@pytest.fixture(scope="session")
+# fetch browser kind, base url and writes a dictionary of key values into allure's environment.properties file
+def get_testing_properties(request, prep_properties):
```

Comment:
fixture name should be == write_allure_enviorment

Reply:
done